exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,12 @@
 var name = "Daniel";
 var danielsRole = "mentor";
 
+if (danielsRole === "mentor") {
```

Comment:
`if (danielsRole)`

that works too. remember you've already defined danielsRole
just wanted to share that otherwise, everything else looks pretty

Reply:
Thanks never thought  of that in that regard....